NSSCodeForNashville/NashvilleHistory
# Ddw marker cards

## Status
**READY/IN DEVELOPMENT**

## Documentation

[ X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description

This pull request finds the user's location and the historic markers, listed from closest to furthest, within a certain radius of the user. The Historic Markers within the given radius are listed below the map. 

## Resolves Issue Number

#15 

## Deploy Notes

```sh
git pull --prune
git checkout ddw-marker-cards
ionic serve --lab
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Make sure the application shows a green marker where you are and lists the historic markers from closest to furthest below the map. 
1. In the MarkersCtrl.js file you can change the radius passed into the getMarkersInRadius function (line 59) to make sure the functionality works for multiple radii

## Impacted Files in Application
List general components of the application that this PR will affect:

* MarkerCtrl.js
* MarkerCardsFact.js
* markers.html
